src/pages/student_detail/StudentDetail.tsx

```diff
@@ -1,26 +1,44 @@
-import { HeaderProperty, InformationContainer, Label, MainContainer, NotFoundStudentMessage, PaymentInfo, PaymentsContainer, PaymentsTable, Row, StudentInfo, StudentInfoContainer, SubTitle, TableBody, TableHeader, Title } from './StudentDetail.styles';
-import { formatCurrency } from '../../utils/Formatter';
-import { useLocation } from 'react-router';
-import { useGetStudentByIdQuery } from '../../app/services/StudentService';
-import CheckIcon from '../../assets/check.webp'
+import {
+  HeaderProperty,
+  InformationContainer,
+  Label,
+  MainContainer,
+  NotFoundStudentMessage,
+  PaymentInfo,
+  PaymentsContainer,
+  PaymentsTable,
+  Row,
+  StudentInfo,
+  StudentInfoContainer,
+  SubTitle,
+  TableBody,
+  TableHeader,
+  Title,
+} from "./StudentDetail.styles";
+import { formatCurrency } from "../../utils/Formatter";
+import { useGetStudentByIdQuery } from "../../app/services/StudentService";
+import CheckIcon from "../../assets/check.webp";
+import { useParams } from "react-router-dom";
 
 const StudentDetail = () => {
+  const { studentId } = useParams<{ studentId: string }>();
+  const { data: student, isError } = useGetStudentByIdQuery(studentId!);
 
-  const { studentId } = useLocation().state;
-  const { data: student, isError } = useGetStudentByIdQuery(studentId)
-
-  if (isError || !student) return (
-    <div>
-      <NotFoundStudentMessage>
-        Ocurrió un error al buscar la información del alumno. <br />
-        Intente nuevamente
-      </NotFoundStudentMessage>
-    </div>
-  )
+  if (isError || !student)
```

Comment:
Me parece excelente mostrar un mensaje de error al no encontrar el estudiante :)

Lo único, ojo con esta condicion porque en un instante 0, el usuario no existe (la request se está haciendo al backend), por lo que vas a estrar mostrando el mensaje y es incorrecto


```suggestion
  if (isError)
```

Reply:
Okis, igual de eso no modifiqué nada jajaja, quedó lo que habías hecho cuando hiciste lo del detalle del alumno

Reply:
<img width="296" height="280" alt="image" src="https://github.com/user-attachments/assets/d71ddefc-c711-4a39-b605-ffe56357abc5" />